app/models/spree/gateway/mollie_gateway.rb

```diff
@@ -72,9 +72,13 @@ def prepare_payment_params(money_in_cents, source, gateway_options)
       order_number = gateway_options[:order_id]
       customer_id = gateway_options[:customer_id]
       amount = money_in_cents / 100.0
+      currency = gateway_options[:currency]
 
       order_params = {
-          amount: amount,
+          amount: {
+              value: sprintf("%.2f", amount.to_f),
```

Comment:
This might be an issue for `JPY`?

Reply:
Thanks, yes it is. I'm now using the money object to always use the correct formatting.